Resources/Private/BackendModule/src/Model/ModuleItem.ts

```diff
@@ -0,0 +1,4 @@
+import {FocusKeywordModuleItem} from "./FocusKeywordModuleItem";
+import {AssetModuleItem} from "./AssetModuleItem";
+
+export interface ModuleItem extends AssetModuleItem, FocusKeywordModuleItem {}
```

Comment:
Wenn man im BackendModule is, dann muss vermutlich nicht überall "Module" dranstehen.